README.md

```diff
@@ -42,14 +44,11 @@ Create the configuration file for app runtime.
 
 Expose configuration file's path as an environment variable in `AUTH_APP_CONFIG_PATH`.
 
-***Important Note:*** The environment where the application will be deployed, the timezone MUST be as UTC. Also, make sure redis is version `6.*`
+***Important Note:*** Make sure redis is version `7.*`
 
-### Architecture (TODO: OUTDATED)
+### Architecture
```

Comment:
Could you plz add some intro to the README about how this service is used?
Is it basically for KDF app to access backends (web3, quicknode etc providers)? 
Will it be a network of proxies or each KDF peer will have its own proxy locally?

Reply:
> Could you plz add some intro to the README about how this service is used?

Sure.

> Is it basically for KDF app to access backends (web3, quicknode etc providers)?

Yes, we use this to hide the keys for our paid services and protect them from misuse. You can take a look on https://gist.github.com/onur-ozkan/725f56c05a2eb01e0428f03191f8da86 for some more context.

> Will it be a network of proxies or each KDF peer will have its own proxy locally?

No.

Reply:
> Could you plz add some intro to the README about how this service is used?

[Done](https://github.com/KomodoPlatform/komodo-defi-proxy/pull/25/commits/9ba90c1690e6568ca59a134f99287fbae0954088) that.